roles/sap_ha_pacemaker_cluster/tasks/construct_vars_nwas_abap_ascs_ers.yml

```diff
@@ -391,10 +391,10 @@
   vars:
     __constraint_order_sapmnt:
       resource_first:
-        id: "{{ sap_ha_pacemaker_cluster_nwas_sapmnt_filesystem_resource_name }}-clone"
+        id: "cln_{{ sap_ha_pacemaker_cluster_nwas_sapmnt_filesystem_resource_name }}"
```

Comment:
Do we need vars for the clone names too or is this a typo?
Because the resource would not exist as `cln_rsc_fs_...` when this constraint is created.

Reply:
Thank you @ja9fuchs it is resolved in latest commit.